src/components/Map.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react"
```

Comment:
https://github.com/codechrysalis/cc2-project.continuous-delivery/tree/master/src

Instead of using component directory, Could you use container? (react-redux)
If you will create container from next task please let me know:)

Reply:
@duyoji 
I will create a container for the next task.
So the next task is to change current Map component to react-redux one?
Thanks :)

Reply:
> I will create a container for the next task.

I got it!